src/applications/appeals/10182/tests/config/submit-transformer.unit.spec.js

```diff
@@ -31,7 +58,7 @@ describe('transform', () => {
   });
 
   it('should transform maximal-test.json with part3 data correctly', () => {
-    const data = { data: { [SHOW_PART3]: true, ...maximalData.data } };
+    const data = { data: { ...maximalData.data, [SHOW_PART3]: true } };
```

Comment:
Did the order through this one off?

Reply:
yes, any data in the first entry is replaced by data from the second, third, etc...